src/index.js

```diff
@@ -34,11 +35,14 @@ export function call(obj, args, cons = console){
     }
 }
 
-export function run(cmd, options){
-    options = options || {};
+export function run(cmd, options = {}){
+    options.env  = options.env || {};
+    options.env.PATH = [
+      `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+    ].join(path.delimiter);
```

Comment:
Yup, my mistake I was thinking about making use of `path.sep` so actually it could like this:

```
options.env.PATH = [
 [process.cwd(), 'node_modules', '.bin'].join(path.sep), process.env.PATH
].join(path.delimiter);
```